server/src/main/java/keywhiz/commands/BackfillOwnershipCommand.java

```diff
@@ -33,17 +33,19 @@ public BackfillOwnershipCommand() {
   }
 
   @Override public void configure(Subparser subparser) {
+    super.configure(subparser);
+
     subparser.addArgument("--batchSize")
         .dest(Args.BATCH_SIZE)
         .type(Integer.class)
         .setDefault(1)
-        .help("Number of records to process in one batch (default is 1)");
+        .help("Number of records to process in one batch");
 
     subparser.addArgument("--delay")
         .dest(Args.DELAY)
         .type(String.class)
         .setDefault("PT0S")
-        .help("Delay between batches in ISO 8601 duration format e.g. PT0.1s (default is no delay)");
+        .help("Delay between batches in ISO 8601 duration format");
   }
```

Comment:
It turns out that DropWizard's built-in command help system automatically prints out the default value for an argument so there's no need to provide it manually.